cypress/e2e/counters.cy.js

```diff
@@ -150,93 +150,95 @@ describe('Counters CRUD', () => {
         cy.get("[aria-label='counters-table-label']").contains("td", "strong-5").should("not.exist");
       });
 
-      it ('successfully filters counters', () => {
```

Comment:
@andyuk1986 the behaviour works, but this test stops because is unable to open "the filter". i'm under the impression there is a bug in patternly when doing the responsive design behavior ... do you have time to check for this? (you can say not now and I'll log a jira)